docs/cloud-native-security/vuln-management-overview.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[vuln-management-overview]]
+= Cloud native vulnerability management
+
+Elastic's Cloud Native Vulnerability Management (CNVM) feature helps you identify known vulnerabilities in your cloud workloads.
+
+Setup uses infrastructure as code. For instructions, refer to <<vuln-management-get-started, Get started with Cloud Native Vulnerability Management>>.
+
+NOTE: CNVM currently only supports AWS EC2 Linux workloads.
+
+.Requirements
+[sidebar]
+--
+* CNVM is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* {stack} and {agent} versions 8.8 or higher.
```

Comment:
This needs to be a complete sentence, like the rest of the list. (I know the sidebar is called "Requirements" so this is implied, but this is more for consistency.)
```suggestion
* Requires {stack} and {agent} version 8.8 or higher.
```

Reply:
@benironside @joepeeples I think historically, we use `earlier` and `later` when referring to stack versions. I think this was a UX writing question at one point, but I don't think we should use `higher`. Of course I went down a rabbit hole and found we use various forms of this in our docs... 🙃